modules/quick-start-components.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * web_console/creating-quick-start-tutorials.adoc
+
+[id="quick-start-components_{context}"]
+= Quick start components
+
+A quick start consists of these sections.
+
+* *Card*: The catalog tile that provides the quick start’s basic information, including title, description, time commitment, and completion status.
+* *Introduction*: A brief overview of the quick start’s goal and tasks.
+* *Task headings*: Hyper-linked titles for each task in the quick start.
+* *Check your work module*: A module for a user to confirm that they completed a task successfully before advancing to the next task in the quick start.
+* *Hints*: An animation to help users identify specific areas of the product.
+* *Buttons*
+** *Next and back buttons*: Buttons for navigating the steps and modules within each task of a quick start.
+** *Final screen buttons*: Buttons for closing the quick start, going back to previous tasks within the quick start, and viewing all quick starts.
+
+The main content area of a quick start are:
+
+* *Card copy*
+* *Introduction*
+* *Task steps*
+* *Modals and in-app messaging*
+* *Check your work module*
```

Comment:
It would be better if we can add some docs/steps about how to draft `Task steps` and `Check your work module` 

We can add them to `Mapping the elements in the quick start to the quick start CR`

Reply:
Good idea! We will be expanding this content post-4.7 and will add that information in then. Thank you!